artifactory/services/releasebundle.go

```diff
@@ -10,12 +11,14 @@ import (
 	"github.com/jfrog/jfrog-client-go/utils/io/fileutils"
 	"github.com/jfrog/jfrog-client-go/utils/log"
 	"net/http"
+	"strings"
 )
 
 const (
-	conflictErrorMessage               = "Bundle already exists"
-	ReleaseBundleImportRestApiEndpoint = "api/release/import/"
-	octetStream                        = "application/octet-stream"
+	conflictErrorMessage                    = "Bundle already exists"
+	ReleaseBundleImportRestApiEndpoint      = "api/release/import/"
+	octetStream                             = "application/octet-stream"
+	ReleaseBundleExistInRbV2RestApiEndpoint = "lifecycle/api/v2/release_bundle/existence"
```

Comment:
As this is a lifecycle endpoint, And RBV2 is under "lifecycle", let's move the relevant code to lifecycle-services.

It is indeed confusing because this is the code for V1.